src/lmnt/api.py

```diff
@@ -95,12 +123,79 @@ async def synthesize(self, text, voice, **kwargs):
     form_data = aiohttp.FormData()
     form_data.add_field('text', text)
     form_data.add_field('voice', voice)
-    form_data.add_field('seed', kwargs.pop('seed', 0))
-    form_data.add_field('format', kwargs.pop('format', 'wav'))
-    form_data.add_field('speed', kwargs.pop('speed', 1.0))
+    form_data.add_field('seed', kwargs.get('seed', 0))
+    form_data.add_field('format', kwargs.get('format', 'wav'))
+    form_data.add_field('speed', kwargs.get('speed', 1.0))
+
+    is_multipart_response = False
+    extras = ''
+
+    has_duration = kwargs.get('durations', False)
```

Comment:
I think I would go with "alignment" instead of "durations"?

Reply:
Done.